packages/block-editor/src/store/selectors.js

```diff
@@ -2943,9 +2944,13 @@ export const getBlockEditingMode = createRegistrySelector(
 					state,
 					sectionRootClientId
 				);
-				if ( ! sectionsClientIds?.includes( clientId ) ) {
-					return 'disabled';
+
+				// Sections are always contentOnly.
+				if ( sectionsClientIds?.includes( clientId ) ) {
+					return 'contentOnly';
 				}
```

Comment:
While it might set the section to `contentOnly` block editing mode, the inner content blocks don't seem to be editable when I test it.

`contentOnly` block editing mode is different to `contentOnly` template lock.  The block editing mode only affects the current client id and not the inner blocks of the current client id.

To make the content blocks `contentOnly` I think you'll need to use `__experimentalHasContentRoleAttribute` like the code a few lines below this to determine whether a block is a content block.

Reply:
If I'm understanding you correctly (please say if I'm not) then that is exactly what I want to happen in this PR.

In "zoom out" the intent is that only the sections themselves are editable (for now) and not the inner blocks.

If other efforts around content only don't land, then we might want to adjust zoom out behaviour to do what you suggest. 

Let me know if thats confusing or something seems off.

Reply:
Ah, ok, I have the wrong end of the stick then. I thought the goal would be to make content editable in zoom out. I guess the advantage of using contentOnly in the way this PR does it to reduce the number of inspector controls?

Reply:
Yes that's right. 

In the future we anticipate making all the "content" blocks editable but for now we are sticking with only being able to select top level "sections".

> ....to reduce the number of inspector controls?

Yes that's the idea. If you select a section you shouldn't be overwhelmed with layout options on the section.